#10069 add new suggestion template

DS-INRA opened the pull request:
**What this PR does / why we need it**:
Add a new issue template for suggestion to distinguish them from feature request at creation

**Which issue(s) this PR closes**:
- Closes #10069

**Additional documentation**:
Previous similar issue : 
- #10081

sekmiller commented on .github/ISSUE_TEMPLATE/idea_proposal.md:
```diff
@@ -0,0 +1,40 @@
+Start below this comment section.
+-->
+
+**Overview of the Feature Request**
```
Do we want to call this "Idea Suggestion" , not "Feature Request" here?

sekmiller commented on .github/ISSUE_TEMPLATE/idea_proposal.md:
```diff
@@ -0,0 +1,40 @@
+**Overview of the Feature Request**
+
+
+**What kind of user is the feature intended for?**
```
Again, "suggestion" for "feature"?

sekmiller commented on .github/ISSUE_TEMPLATE/idea_proposal.md:
```diff
@@ -0,0 +1,40 @@
+**Any brand new behavior do you want to add to Dataverse?**
+
+
+**Any open or closed issues related to this feature request?**
```
again, "idea or suggestion"?

sekmiller commented:
@DS-INRA Dimitri, it looks like there are cut and paste issues with the body of the template. Please take a look. Thanks!

pdurbin commented on .github/ISSUE_TEMPLATE/idea_proposal.md:
```diff
@@ -0,0 +1,40 @@
+<!--
+Thank you for contributing to the Dataverse Project through the creation of a feature request!
+
+More information on ideas/feature requests and contributions can be found in the "Contributing to Dataverse" page:
+https://github.com/IQSS/dataverse/blob/develop/CONTRIBUTING.md#ideasfeature-requests
```
Heads up that in this PR we plan to move content from CONTRIBUTING.md to its own guide:

- #10532

So that #ideasfeature-requests link won't work anymore.

pdurbin reviewed (commented):
I'm just raising awareness of the upcoming Contributor Guide.

sekmiller commented:
@DS-INRA Dimitri when you have some time please let me know what you think about my comments/questions above. Thanks!

DS-INRA commented:
> @DS-INRA Dimitri when you have some time please let me know what you think about my comments/questions above. Thanks!

Hello, sorry for the long delay, I should have more time to get back on this after this week :)

DS-INRA commented:
Thanks for the reviews!
I changed the "feature" mentions and updated the contributor's guide link (including in existing issue templates)
I also added the section:
```
**Will you be able to contribute a Pull Request for this issue?**  
Help is always welcome, is this bug something you or your organization plan to fix ?
```
to the existing issue templates, but can take it back if deemed irrelevant :)

sekmiller reviewed (approved):
Thanks for the updates, Dimitri. Looks good!

pdurbin commented on .github/ISSUE_TEMPLATE/bug_report.md:
```diff
@@ -53,3 +52,6 @@ To add a screenshot, please use one of the following formats and/or methods desc
 * https://help.github.com/en/articles/file-attachments-on-issues-and-pull-requests
 *
+
+**Will you be able to contribute a Pull Request for this issue?**  
```
```suggestion
**Will you be able to contribute a pull request for this issue?**  
```

DS-INRA replied:
changed with the last comment's rephrasing

pdurbin commented on .github/ISSUE_TEMPLATE/bug_report.md:
```diff
@@ -53,3 +52,6 @@ To add a screenshot, please use one of the following formats and/or methods desc
 *
+
+**Will you be able to contribute a Pull Request for this issue?**  
+Help is always welcome, is this bug something you or your organization plan to fix ?
```
```suggestion
Help is always welcome, is this bug something you or your organization plan to fix?
```

pdurbin commented on .github/ISSUE_TEMPLATE/feature_request.md:
```diff
@@ -34,3 +33,6 @@ Start below this comment section.
 
 **Any open or closed issues related to this feature request?**
+
+**Will you be able to contribute a Pull Request for this issue?**  
+Help is always welcome, is this bug something you or your organization plan to fix ?
```
```suggestion
**Will you be able to contribute a pull request for this issue?**  
Help is always welcome, is this bug something you or your organization plan to fix?
```

DS-INRA replied:
changed with the last comment's rephrasing

pdurbin commented on .github/ISSUE_TEMPLATE/idea_proposal.md:
```diff
@@ -0,0 +1,40 @@
+**Any open or closed issues related to this suggestion?**
+
+
+**Will you be able to contribute a Pull Request for this issue?**  
+Help is always welcome, is this suggestion something you or your organization plan to implement ?
```
```suggestion
**Will you be able to contribute a pull request for this issue?**  
Help is always welcome, is this suggestion something you or your organization plan to implement?
```

DS-INRA replied:
changed with the last comment's rephrasing

pdurbin commented on .github/ISSUE_TEMPLATE/feature_request.md:
```diff
@@ -11,8 +11,7 @@ assignees: ''
 Thank you for contributing to the Dataverse Project through the creation of a feature request!
```
Above, should we change

```
title: 'Feature Request/Idea:'
```

to just

```
title: 'Feature Request:'
```

? That is, remove "idea" now that we have a dedicate template for ideas?

DS-INRA replied:
Nice catch! I removed "Idea"

pdurbin commented on .github/ISSUE_TEMPLATE/bug_report.md:
```diff
@@ -53,3 +52,6 @@ To add a screenshot, please use one of the following formats and/or methods desc
 
 * https://help.github.com/en/articles/file-attachments-on-issues-and-pull-requests
 *
+
```
At a high level, "Will you be able to contribute a pull request for this issue?" strikes me as somewhat strong.

What if they say "yes" but it doesn't work out? Will they feel bad?

Instead should we say something softer like "Are you thinking about creating a pull request for this issue?"

That way they won't feel as "on the hook", I would hope.

DS-INRA replied:
Sure, it does sound less intimidating

pdurbin reviewed (commented):
Just some thoughts.

DS-INRA commented:
@pdurbin I have updated the templates according to all the comments :slightly_smiling_face:
